### DISCUSSION TITLE
Customize DNSBL sites

### DISCUSSION THREAD

**Discussion by infernusec** [docker-mailserver/docker-mailserver#4215] [General]
Hey everyone,
i want to add more RBL checks, but as i can see, the main.cf has hardcoded the following servers:
```
postscreen_dnsbl_sites =
    zen.spamhaus.org=127.0.0.[2..11]*3
    bl.mailspike.net=127.0.0.[2;14;13;12;11;10]
    b.barracudacentral.org*2
    bl.spameatingmonkey.net=127.0.0.2
    dnsbl.sorbs.net
    psbl.surriel.com
    list.dnswl.org=127.0.[0..255].0*-2
    list.dnswl.org=127.0.[0..255].1*-3
    list.dnswl.org=127.0.[0..255].[2..3]*-4
```

how can i add more servers?

**Comment by casperklein** [comment]
You can add your own config to `main.cf`. See [here](https://docker-mailserver.github.io/docker-mailserver/v14.0/config/advanced/override-defaults/postfix/).
